test/controllers/EditSetRecordControllerSpec.scala

```diff
@@ -1731,8 +1731,8 @@ class EditSetRecordControllerSpec extends BaseControllerSpec {
     */
```

Comment:
I would amend this comment to read:

> The actual list has no relevance to these tests, at least until we validate the legal status ID upon submission.

And - if the ticket exists for doing this validation - it would good to add it.

Reply:
Comment updated but no ticket at the moment.

Reply:
Confirmed.